### PR TITLE
Fix Xvfb issue and go back to Ubuntu 20.04

### PR DESCRIPTION
**Pull request by cdietrich**

### CODE REVIEW

File: integrationtests/org.eclipse.xtext.swtbot.testing/src/org/eclipse/xtext/swtbot/testing/api/EclipseAPI.java [hunk 1]
```diff
@@ -35,6 +37,12 @@ private EclipseAPI() {
 	}
 
 	public static MainMenuAPI mainMenu() {
+		UIThreadRunnable.syncExec(new VoidResult() {
```
Comment (cdietrich) [inline integrationtests/org.eclipse.xtext.swtbot.testing/src/org/eclipse/xtext/swtbot/testing/api/EclipseAPI.java:40]: why is this needed? is there a better place?

Comment (ArneDeutsch) [reply integrationtests/org.eclipse.xtext.swtbot.testing/src/org/eclipse/xtext/swtbot/testing/api/EclipseAPI.java:40]: Currently executing the tests again and noticed that they do not run in full-screen-mode ... maybe the code in AbstractSwtBotTest is not executed for som ereason ...

Comment (ArneDeutsch) [reply integrationtests/org.eclipse.xtext.swtbot.testing/src/org/eclipse/xtext/swtbot/testing/api/EclipseAPI.java:40]: Guess instead of this we have to fix the usage of junit here ... remove the @BeforeClass in the abstract superclass and explicitely call it from subclass ...

Comment (szarnekow) [reply integrationtests/org.eclipse.xtext.swtbot.testing/src/org/eclipse/xtext/swtbot/testing/api/EclipseAPI.java:40]: Quoting the JavaDoc of BeforeClass:

> The <code>@BeforeClass</code> methods of superclasses
>  will be run before those of the current class, unless they are shadowed in the current class.
 
 Not sure if that's the real issue.

**Comment by cdietrich** [comment]
replaced with https://github.com/itemis/xtext-reference-projects/pull/171

**Comment by cdietrich** [comment]
replaced with https://github.com/itemis/xtext-reference-projects/pull/171
